### PR TITLE
13.0 staging qdp

### PR DESCRIPTION
**Pull request by qdp-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_move.py [hunk 1]
```diff
@@ -3626,8 +3626,8 @@ def _check_reconcile_validity(self):
         for line in self:
             company_ids.add(line.company_id.id)
             all_accounts.append(line.account_id)
-            if (line.matched_debit_ids or line.matched_credit_ids) and line.reconciled:
-                raise UserError(_('You are trying to reconcile some entries that are already reconciled.'))
+            if line.reconciled:
+                raise UserError(_('You are trying to reconcile some entries that are already reconciled!'))
```
Comment (MiquelRForgeFlow) [inline addons/account/models/account_move.py]: then update `.pot` file, but I was told this kind of change cannot be done in stable :S

Comment (qdp-odoo) [reply addons/account/models/account_move.py]: ho good Lord -_- 

the point was to change the condition, not the punctuation of the error message. I didn't even noticed it changed... let me check that

Comment (qdp-odoo) [reply addons/account/models/account_move.py]: fixed!

thanks for noticing :+1:

File: addons/account/models/account_move.py [hunk 2]
```diff
@@ -2825,7 +2825,7 @@ def _get_fields_onchange_balance_model(self, quantity, discount, balance, move_t
                 'price_unit': balance / (quantity or 1.0),
             }
         else:
-            vals = {}
+            vals = {'price_unit': 0.0}
```
Comment (smetl) [inline addons/account/models/account_move.py]:
```
elif discount_factor != 100.0:
  vals = {'price_unit': 0.0}
else:
  vals = {}
```

Comment (qdp-odoo) [reply addons/account/models/account_move.py]: I don't get the reason why?

Comment (qdp-odoo) [reply addons/account/models/account_move.py]: got it

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**